hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -0,0 +1,172 @@
+          desc: "The count at which a wakeup interrupt should be generated",
+        }
+      ]
+    },
+    { name: "WKUP_COUNT",
+      desc: "Wakeup Timer Count Register",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      fields: [
+        { bits: "31:0",
+          name: "count",
+          desc: "The current wakeup counter value",
+        }
+      ]
+    },
+    { name: "WDOG_REGWEN",
+      desc: "Watchdog Timer Write Enable Register",
+      swaccess: "rw1c",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "regwen",
+          desc: "Once cleared, the watchdog configuration will be locked until the next reset",
+          resval: 1
+        }
+      ]
+    },
+    { name: "WDOG_CTRL",
+      desc: "Watchdog Timer Control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "WDOG_REGWEN",
+      fields: [
+        { bits: "0",
+          name: "enable",
+          desc: "When set to 1, the watchdog timer will count",
+        },
+        { bits: "1",
+          name: "pause_in_sleep",
+          desc: "When set to 1, the watchdog timer will not count during sleep",
+        }
+      ]
+    },
+    { name: "WDOG_BARK_THOLD",
+      desc: "Watchdog Timer Bark Threshold Register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "WDOG_REGWEN",
+      fields: [
+        { bits: "31:0",
+          name: "threshold",
+          desc: "The count at which a watchdog bark interrupt should be generated",
+        }
+      ]
+    },
+    { name: "WDOG_BITE_THOLD",
+      desc: "Watchdog Timer Bite Threshold Register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "WDOG_REGWEN",
+      fields: [
+        { bits: "31:0",
+          name: "threshold",
+          desc: "The count at which a watchdog bite reset should be generated",
+        }
+      ]
+    },
+    { name: "WDOG_COUNT",
+      desc: "Watchdog Timer Count Register",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      fields: [
+        { bits: "31:0",
+          name: "count",
+          desc: "The current watchdog counter value",
+        }
+      ]
+    },
+    { name: "INTR_STATE",
+      desc: "Interrupt State Register",
+      swaccess: "rw1c",
+      hwaccess: "hrw",
+      fields: [
+        { bits: "0",
+          name: "wkup_timer_expired",
+          desc: "Raised if the wakeup timer has hit the specified threshold",
+        }
+        { bits: "1",
+          name: "wdog_timer_expired",
+          desc: "Raised if the watchdog timer has hit the bark threshold",
+        }
+      ]
+    },
+    { name: "INTR_TEST",
+      desc: "Interrupt Test Register",
+      swaccess: "wo",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "wkup_timer_expired",
+          desc: "Write 1 to force wkup_timer_expired interrupt",
+        }
+        { bits: "1",
+          name: "wdog_timer_expired",
+          desc: "Write 1 to force wdog_timer_expired interrupt",
```

Comment:
I think you should add that this is `bark` interrupt the same you have done in the `INTR_STATE` register description.

Reply:
Will do